ps_imageslider.php

```diff
@@ -583,8 +590,9 @@ protected function updateUrl($link)
             return $link;
         }
 
-        if (substr($link, 0, 7) !== 'http://' && substr($link, 0, 8) !== 'https://') {
-            $link = 'http://' . $link;
+        $url = parse_url($link);
+        if (empty($url['host'])) {
+            $link = $this->context->link->getBaseLink() . ltrim($link, '/');
```

Comment:
This is the fix. The rest is automatic refacto.

Reply:
with $link = 'www.prestashop.com':
$url['host'] is empty while $url['path'] = 'www.prestashop.com'

Reply:
I know and I have fix for that ready. Will add it soon...